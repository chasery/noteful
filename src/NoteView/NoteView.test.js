import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import NoteView from "./NoteView";

describe("NoteView component", () => {
  it("renders without error", () => {
    const wrapper = shallow(<NoteView />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
