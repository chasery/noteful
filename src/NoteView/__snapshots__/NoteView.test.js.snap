// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoteView component renders without error 1`] = `
<div>
  NoteView
</div>
`;
